Pulsarc/UI/Screens/Gameplay/HitObject.cs

```diff
@@ -139,24 +138,22 @@ protected virtual double CalcZLocation(int currentTime, double speed, double cro
         /// <param name="crosshairZLoc">Current crosshair Z-Location</param>
         protected virtual void HandleHiddenState(double crosshairZLoc)
         {
-            if (Hidden)
-            {
-                // When arcs are fully hidden, currently at the 2/3rds mark (between first being seen and being hit)
-                double fullFadeLocation = crosshairZLoc - (crosshairZLoc / 3);
-
-                // New opacity is calculated by looking at how far the arc has gone.
-                float newOpacity = (float)(fullFadeLocation - ZLocation) / (float)(fullFadeLocation);
-
-                if (ZLocation > fullFadeLocation)
-                    newOpacity = 0f;
-                // This is for playing with Hidden. Zooms can make 
-                // the arcs more opaque with the current implementation
-                // of hidden. This makes sure that arcs don't gain opacity.
-                else if (newOpacity > Opacity)
-                    newOpacity = Opacity;
-
-                Opacity = newOpacity;
-            }
+            if (!Hidden) return;
```

Comment:
In most places you've written oneline if statement like this:
```
if (blabla)
   // Do something
```

Perhaps try to keep this consistent all over the codebase?

Reply:
this is an inverse if statement, if something is false it doesnt let it continue, it is for reducing nesting